autoload/neoformat/formatters/ruby.vim

```diff
@@ -8,3 +8,11 @@ function! neoformat#formatters#ruby#rubybeautify() abort
         \ 'args': ['--spaces', '-c ' . shiftwidth()],
         \ }
 endfunction
+
+function! neoformat#formatters#ruby#rubocop() abort
+     return {
+        \ 'exe': 'rubocop',
+        \ 'args': ['--auto-correct', '--stdin', '-', '2>/dev/null', '|', 'sed "1,/^====================$/d"'],
```

Comment:
`'-'` could be changed to `'%'` so that `expand()` can be used to get the filename